tests/OmniSharp.Tests/FixUsingsFacts.cs

```diff
@@ -0,0 +1,348 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OmniSharp.Models;
+using Xunit;
+ 
+namespace OmniSharp.Tests
+{
+    public class FixUsingsFacts
+    {
+
+        string fileName = "test.cs";
+
+        [Fact]
+        public async Task FixUsings_AddsUsingSingle()
+        {
+            const string fileContents = @"namespace nsA {
+    public class classX{}
+}
+
+namespace OmniSharp {
+    public class class1 
+    {
+        public method1()
+        {
+            var c1 = new classX();
+        }
+    }
+}";
+            string expectedFileContents = @"using nsA;" + 
+            Environment.NewLine +
+@"namespace nsA {
+    public class classX{}
+}
+
+namespace OmniSharp {
+    public class class1 
+    {
+        public method1()
+        {
+            var c1 = new classX();
+        }
+    }
+}";
+
+            await AssertBufferContents(fileContents, expectedFileContents);
+        }
+
+        [Fact]
+        public async Task FixUsings_AddsUsingSingleForFrameworkMethod()
+        {
+            const string fileContents = @"namespace OmniSharp {
+    public class class1 
+    {
+        public void method1()()
+        {
+            Console.WriteLine(""abc"");
+        }
+    }
+}";
+            string expectedFileContents = @"using System;"  + 
+            Environment.NewLine + 
+@"namespace OmniSharp {
+    public class class1 
+    {
+        public void method1()()
+        {
+            Console.WriteLine(""abc"");
+        }
+    }
+}";
+
+            await AssertBufferContents(fileContents, expectedFileContents);
+        }
+
+        [Fact]
+        public async Task FixUsings_AddsUsingSingleForFrameworkClass()
+        {
+            const string fileContents = @"namespace OmniSharp {
+    public class class1 
+    {
+        public void method1()()
+        {
+            var s = new StringBuilder();
+        }
+    }
+}";
+            string expectedFileContents = @"using System.Text;"  + 
+            Environment.NewLine + 
```

Comment:
You shouldn't be using Environment.NewLine in either place. It should get the newline character from the configuration.